plugins/templates/js/yangdoc.menu.js

```diff
@@ -0,0 +1,87 @@
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+//
+//
+// Obtained from bootsnipp.com
+// http://bootsnipp.com/SeanWessell/snippets/ypNAe
+
+$.fn.extend({
+    treed: function (o) {
+
+      var openedClass = 'glyphicon-minus-sign';
+      var closedClass = 'glyphicon-plus-sign';
+
+      if (typeof o != 'undefined'){
+        if (typeof o.openedClass != 'undefined'){
+        openedClass = o.openedClass;
+        }
+        if (typeof o.closedClass != 'undefined'){
+        closedClass = o.closedClass;
+        }
+      };
+
+    //initialize each of the top levels
+    var tree = $(this);
+    tree.addClass("tree");
+    tree.find('li').has('ul').each(function () {
+        var branch = $(this); //li with children ul
+        branch.prepend("<i class='indicator glyphicon " + closedClass + "'></i>");
+        branch.addClass('branch');
+        branch.on('click', function (e) {
+            if (this == e.target) {
+                var icon = $(this).children('i:first');
+                icon.toggleClass(openedClass + " " + closedClass);
+                $(this).children().children().toggle();
+            }
+        })
+        branch.children().children().toggle();
+    });
+    //fire event from the dynamically added icon
+  tree.find('.branch .indicator').each(function(){
+    $(this).on('click', function () {
+        $(this).closest('li').click();
+    });
+  });
+    //fire event to open branch if the li contains an anchor instead of text
+    // tree.find('.branch>a').each(function () {
+    //     $(this).on('click', function (e) {
+    //         $(this).closest('li').click();
+    //         e.preventDefault();
+    //     });
+    // });
+    //fire event to open branch if the li contains a button instead of text
+    tree.find('.branch>button').each(function () {
+        $(this).on('click', function (e) {
+            $(this).closest('li').click();
+            e.preventDefault();
+        });
+    });
+}
+});
+
+
+//Initialization of treeviews
```

Comment:
Should we clean this comment out?

Additional question - have there been modifications to this file (and the menu CSS) that we need to mark with a different copyright?